apps/api/src/hyp3_api/validation.py

```diff
@@ -99,7 +99,7 @@ def check_not_antimeridian(granule_metadata):
         if abs(bbox[0] - bbox[2]) > 180.0 and bbox[0] * bbox[2] < 0.0:
             msg = (
                 f'Granule {granule["name"]} crosses the antimeridian.'
-                ' Processing across the antimeridian is not currently supported.',
+                ' Processing across the antimeridian is not currently supported.'
             )
```

Comment:
```suggestion
            msg = f'Granule {granule["name"]} crosses the antimeridian.' \
                  ' Processing across the antimeridian is not currently supported.'
```

Avoids the comma shenanigans